ceph-extra-exporter/log.go

```diff
@@ -1,10 +1,32 @@
 package main
 
-import "github.com/cybozu-go/log"
+import (
+	"log/slog"
+	"os"
+)
 
-var logger *log.Logger
+var logger *slog.Logger
 
 func init() {
-	logger = log.NewLogger()
-	logger.SetFormatter(log.JSONFormat{})
+	utsname, err := os.Hostname()
```

Comment:
Because you get the value from `os.Hostname()` function, I think the variable name should be `hostname` (though `utsname` has almost the same meaning.)

Reply:
That's right. I'll fix it.